pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -233,6 +233,15 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxWindowsConfig(pod *v1.Pod)
 		SecurityContext: &runtimeapi.WindowsSandboxSecurityContext{},
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(features.WindowsHostNetwork) {
```

Comment:
just curiosity, what is the relation with `features.WindowsHostProcessContainers`?

are all `WindowsHostProcessContainers` HostNetwork pods?

Reply:
Yes, All pods that contain HostProcessContainers must set `hostNetwork=true` (this is enforced at API admission).
Really HostProcessContainers just start processes on the host but they are packages and deployed like contianers (hence the name).

With these changes pods that do not contain HostProcessContainers will have the option of using pod or node network namespace.